android/src/main/java/com/aakashns/reactnativedialogs/modules/DialogAndroid.java

```diff
@@ -163,18 +164,22 @@ else if( tg.equals("end") )
 
     @ReactMethod
     public void show(ReadableMap options, final Callback callback) {
-        mBuilder = new MaterialDialog.Builder(getCurrentActivity());
         try {
+            mBuilder = new MaterialDialog.Builder(getCurrentActivity());
             applyOptions(mBuilder, options);
         } catch (Exception e) {
             callback.invoke("error", e.getMessage(), options.toString());
+            return;
         }
 
         if (options.hasKey("onPositive")) {
             mBuilder.onPositive(new MaterialDialog.SingleButtonCallback() {
                 @Override
                 public void onClick(MaterialDialog materialDialog, DialogAction dialogAction) {
-                    callback.invoke("onPositive");
+                    try {
+                        callback.invoke("onPositive");
+                    } catch (Exception e) {
+                    }
```

Comment:
@faogustavo Thanks for the PR. One thing I would recommend is to add a log statement where ever you are catching such exceptions so that if the callbacks are not getting invoked for any reason, the user could at least see a log in the log cat. A simple `Log.d("react-native-dialogs", "Unable to invoke the onPositive callback. " + e.getMessage());` would suffice.